src/astro/files/types/base.py

```diff
@@ -38,3 +38,6 @@ def name(self):
     def __str__(self):
         """String representation of type"""
         return self.name.value
+
+    def __repr__(self):
+        return f'{self.__class__.__name__}(path="{self.path}",conn_id="{self.normalize_config}")'
```

Comment:
`normalize_config` is optional so just path would be good

```suggestion
        return f'{self.__class__.__name__}(path="{self.path}")'
```